pkg/status/status.go

```diff
@@ -33,6 +33,8 @@ const (
 	OUTPUT_DETECTION     WorkflowStep = "OUTPUT_DETECTION"
 	BRANCH_CHECKOUT      WorkflowStep = "BRANCH_CHECKOUT"
 	ARTEFACT_UPLOAD      WorkflowStep = "ARTEFACT_UPLOAD"
+	IOS_BUILD            WorkflowStep = "IOS_BUILD"
+	COCOAPODS_INSTALL    WorkflowStep = "COCOAPODS_INSTALL"
```

Comment:
Are we keeping `COCOAPOD_INSTALL` as a separate step? 
Asking this because we usually do not keep deps install as a different step in other stacks

Reply:
haan i have made it in vm a point to have cocoapods already installed, but keeping it just in the case we need to provide user option to change cocoapods version in a particular worklow, we may or may not use this step later.

Reply:
Alright